tests/pages/MainToolbarPage.ts

```diff
@@ -0,0 +1,261 @@
+import { Page } from '@playwright/test';
+
+export class MainToolbarPage {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  get layout() {
+    const page = this.page;
+
+    const layout = {
+      button: page.getByTestId('Layout'),
+      async click() {
+        await page.getByTestId('Layout').click();
+      },
+    };
+
+    return {
+      ...layout,
+      get axialPrimary() {
+        return {
+          button: page.getByTestId('Axial Primary'),
+          async click() {
+            await layout.click();
+            await page.getByTestId('Axial Primary').click();
```

Comment:
Can we do `this.button.click();` instead?

Reply:
I do like your suggestion, but insead of using the constant name `axialPrimaryButton`, can we simply use `button` instead?